config/default/field.field.node.localgov_subsites_page.localgov_subsites_content.yml

```diff
@@ -178,4 +180,7 @@ settings:
       topic_list_builder:
         weight: -64
         enabled: false
+      webform_response_item:
```

Comment:
Where has this come from? I think it should be removed.

Reply:
Well spotted, thanks